src/components/Layout/Layout.js

```diff
@@ -5,7 +5,7 @@ import Helmet from 'react-helmet';
 import styles from './Layout.module.scss';
 
 export const PureLayout = ({ children, title, description, data }) => {
-  const { author, url: siteUrl } = data.site.siteMetadata;
+  const { url: siteUrl, previewImage } = data.site.siteMetadata;
```

Comment:
lol i didn't notice this but removing `author` here is a bug cuz we need it on line 9 below

Reply:
also need to add the `previewImage` here down below into `LayoutQuery` lol i really did not review this PR at all